test-programs/system-test/tests/test.rs

```diff
@@ -21,15 +26,872 @@ use light_test_utils::{
+        remaining_accounts[inputs_struct.input_compressed_accounts_with_merkle_context
+            [num_inputs - 1]
+            .merkle_context
+            .nullifier_queue_pubkey_index as usize] = AccountMeta {
+            pubkey: env.address_merkle_tree_pubkey,
+            is_signer: false,
+            is_writable: true,
+        };
+        create_instruction_and_failing_transaction(
+            context,
+            payer,
+            inputs_struct,
+            remaining_accounts.clone(),
+            ErrorCode::AccountDiscriminatorMismatch.into(),
+        )
+        .await
+        .unwrap();
+    }
+    Ok(())
+}
+
+fn create_address_test_inputs(
+    env: &EnvAccounts,
+    num_addresses: usize,
+) -> (Vec<NewAddressParams>, Vec<[u8; 32]>) {
+    let mut address_seeds = vec![];
+    for i in 1..=num_addresses {
+        address_seeds.push([i as u8; 32]);
+    }
+
+    let mut new_address_params = vec![];
+    let mut derived_addresses = Vec::new();
+    for (_, address_seed) in address_seeds.iter().enumerate() {
+        new_address_params.push(NewAddressParams {
+            seed: *address_seed,
+            address_queue_pubkey: env.address_merkle_tree_queue_pubkey,
+            address_merkle_tree_pubkey: env.address_merkle_tree_pubkey,
+            address_merkle_tree_root_index: 0,
+        });
+        let derived_address =
+            derive_address(&env.address_merkle_tree_pubkey, address_seed).unwrap();
+        derived_addresses.push(derived_address);
+    }
+    (new_address_params, derived_addresses)
+}
+
+pub async fn failing_transaction_address(
+    context: &mut ProgramTestRpcConnection,
+    // test_indexer: &mut TestIndexer<200, ProgramTestRpcConnection>,
+    payer: &Keypair,
+    env: &EnvAccounts,
+    // num_addresses: usize,
+    inputs_struct: &InstructionDataInvoke,
+    remaining_accounts: Vec<AccountMeta>,
+) -> Result<(), RpcError> {
+    // if num_addresses > 2 {
+    //     panic!("num_output_compressed_accounts should be less than 8");
+    // }
+    // let (mut new_address_params, derived_addresses) =
+    //     create_address_test_inputs(env, num_addresses);
+
+    // let proof_rpc_res = test_indexer
+    //     .create_proof_for_compressed_accounts(
+    //         None,
+    //         None,
+    //         Some(derived_addresses.as_slice()),
+    //         Some(vec![env.address_merkle_tree_pubkey; num_addresses]),
+    //         context,
+    //     )
+    //     .await;
+    // for (i, root_index) in proof_rpc_res.address_root_indices.iter().enumerate() {
+    //     new_address_params[i as usize].address_merkle_tree_root_index = *root_index;
+    // }
+    // let (remaining_accounts, inputs_struct) = create_invoke_instruction_data_and_remaining_accounts(
+    //     &new_address_params,
+    //     &[],
+    //     &[],
+    //     &[],
+    //     &[],
+    //     &[],
+    //     Some(proof_rpc_res.proof),
+    //     None,
+    //     false,
+    // );
+    // inconsistent seed
```

Comment:
A comment or TODO on why this is commented out will be helpful

Reply:
was a leftover